optashift-employee-rostering-shared/src/main/java/org/optaplanner/openshift/employeerostering/shared/tenant/TenantRestService.java

```diff
@@ -55,11 +55,18 @@ Tenant addTenant(
     // ************************************************************************
     // TenantConfiguration
     // ************************************************************************
+    /**
+     * @param id never null
+     * @return never null, the id
+     */
```

Comment:
Consistency: either we javadoc all methods on all RestService interfaces, or we don't. "Do or do not" ;) Hybrid states hurt long term quality.

Given that we swaggerdoc them all - and that's what really matters - I wouldn't bother: simply remove this javadoc.